src/eduid_webapp/phone/views.py

```diff
@@ -177,7 +177,15 @@ def verify(user, code, number):
                              'for user {!r}'.format(number, proofing_user))
 
     db = current_app.proofing_statedb
-    state = db.get_state_by_eppn_and_mobile(proofing_user.eppn, number)
+    state = db.get_state_by_eppn_and_mobile(proofing_user.eppn, number,
+            raise_on_missing=False)
+    if state is None:
+        current_app.logger.debug('Invalid verification code for phone {!r}'
```

Comment:
Rather "Couldn't find proofing state for user {} and number {}"

Reply:
fixed